tekton/cd/pipeline/overlays/robocat/config-artifact-bucket.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-artifact-bucket
+  namespace: tekton-pipelines
+data:
+  bucket.service.account.field.name: BOTO_CONFIG
+  bucket.service.account.secret.key: boto-config
+  bucket.service.account.secret.name: tekton-storage
+  location: s3://tekton-pipelines
```

Comment:
does this mean we're using amazon resources now as well as google? how does one access them, is this bucket open to everyone?

Reply:
No, not using AWS, nor google, that's the whole reason for using Minio. 
Minio is an open source implementation of an object storage that exposes an s3 API.
Minio is deployed in the robot cluster and pipelines can use it as an artifact storage, which is local, efficient and it doesn't require a GCS or AWS account to access.